File: ympa_fsharp_giraffe_server/src/model/ProvideOrderItemIdentifiersRequest.fs
namespace ympa_fsharp_giraffe_server.Model

open System
open System.Collections.Generic
open ympa_fsharp_giraffe_server.Model.OrderItemInstanceModificationDTO

module ProvideOrderItemIdentifiersRequest =

  //#region ProvideOrderItemIdentifiersRequest


  type ProvideOrderItemIdentifiersRequest = {
    Items : OrderItemInstanceModificationDTO[];
  }
  //#endregion

File: src/main/scala/org/openapitools/models/ProvideOrderItemIdentifiersRequest.scala
package org.openapitools.models

import io.circe._
import io.finch.circe._
import io.circe.generic.semiauto._
import io.circe.java8.time._
import ympa_scala_finch_server._
import org.openapitools.models.OrderItemInstanceModificationDTO
import scala.collection.immutable.Seq

/**
 * 
 * @param items Список позиций, требующих маркировки. 
 */
case class ProvideOrderItemIdentifiersRequest(items: Seq[OrderItemInstanceModificationDTO]
                )

object ProvideOrderItemIdentifiersRequest {
    /**
     * Creates the codec for converting ProvideOrderItemIdentifiersRequest from and to JSON.
     */
    implicit val decoder: Decoder[ProvideOrderItemIdentifiersRequest] = deriveDecoder
    implicit val encoder: ObjectEncoder[ProvideOrderItemIdentifiersRequest] = deriveEncoder
}
